src/System.Management.Automation/security/Authenticode.cs

```diff
@@ -52,6 +53,8 @@ public enum SigningOption
     /// </summary>
     internal static class SignatureHelper
     {
+        private static Guid WINTRUST_ACTION_GENERIC_VERIFY_V2 = new Guid("00AAC56B-CD44-11d0-8CC2-00C04FC295EE");
```

Comment:
```suggestion
        private static readonly Guid WINTRUST_ACTION_GENERIC_VERIFY_V2 = new Guid("00AAC56B-CD44-11d0-8CC2-00C04FC295EE");
```

Reply:
Unfortunately this cannot be set to `readonly` as it's technically used as a ref argument for `WinVerifyTrust`. While it's passed in as a reference there is no indication that this is mutated in any way. I could create a copy of this readonly struct locally if that is preferred?